frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bs.ts

```diff
@@ -0,0 +1,68 @@
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+}
+
+export const testName = `test-${Math.random()
+  .toString(36)
+  .replace(/[^a-z]+/g, '')
+  .substr(0, 5)}-bs`;
```

Comment:
We really don't need a random testName. Let's just keep it `test-bucket`. And also define it in the spec file and pass it as an arg to https://github.com/openshift/console/pull/7617/files#diff-1938d8e9017476befb9527466910abf417d7bd63bd70cba279cc80852f13d57fR63

Reply:
why do you want to pass it as an arg when we decided to keep the name constant ? Shall i just type 'test-bucket' for the inside `create-store` ?

Reply:
sure.